net/ddns-scripts/Makefile

```diff
@@ -362,6 +373,15 @@ define Package/ddns-scripts-services/install
 endef
 
 
+define Package/ddns-scripts-utils/install
+	$(INSTALL_DIR) $(1)/usr/lib/ddns
+	$(INSTALL_BIN) ./samples/slaac_sample.sh \
+		$(1)/usr/lib/ddns
+	$(INSTALL_BIN) ./samples/getlocalip_sample.sh \
+		$(1)/usr/lib/ddns
```

Comment:
Could you also prefix the files with `ddns`, and remove `_sample.sh` postfix.
So the following files get installed on the target.
* `ddns-getlocalip`
* `ddns-slaac`

The advantage is it feels more like a normal scripts with out `_sample.sh` and  with the prefix it could be find easier on the system.

Reply:
makes sense! done!